test_app/settings.py

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
yes i agree but the test app could be configured to use the `default` backend instead of yours so `ADMINS` does not need to be defined (except in the test settings)